Allow to build on older automakes. this disables color output on tests,
but leaves the AM_SILENT_RULES intact for automakes which support this.

Signed-off-by: Thilo Bangert <bangert@gentoo.org>

diff -Naur lighttpd-1.4.25.orig/configure.ac lighttpd-1.4.25/configure.ac
--- lighttpd-1.4.25.orig/configure.ac	2009-11-25 10:27:12.000000000 +0100
+++ lighttpd-1.4.25/configure.ac	2009-11-25 10:43:20.000000000 +0100
@@ -8,7 +8,8 @@
 
 AC_CANONICAL_TARGET
 
-AM_INIT_AUTOMAKE([-Wall -Wportability -Wno-override -Werror foreign dist-bzip2 tar-ustar silent-rules color-tests])
+m4_pattern_allow([AM_SILENT_RULES]) 
+AM_INIT_AUTOMAKE([-Wall -Wportability -Wno-override -Werror foreign dist-bzip2 tar-ustar])
 AM_SILENT_RULES
 
 # Checks for programs.
